### ISSUE TITLE
bias=False before BatchNorm layer

### ISSUE THREAD

**Issue by ShoufaChen** [AlexHex7/Non-local_pytorch#14]
Thanks very much for your contribution.

In this [line](https://github.com/AlexHex7/Non-local_pytorch/blob/master/lib/non_local.py#L45-L47):

I think the conv layer should have a `bias=False` before the batch norm layer.

Please tell me if I ignore something?

**Comment by AlexHex7** [comment]
@ShoufaChen Yes, thanks for your report. 
I found the paper only mention weight matrix W. Also there is no bias parameters in formulation. Strictly speaking, it should have bias=False. 

But in my opinion, bias parameters would make the model more flexible. On the other hand, bias parameters which initialed to zero cover the situation where bias=False.

If you are interested in it, you can conduct some experiments to do the comparision. And please share the results with me, thanks.
